src/Microsoft.Azure.NotificationHubs/Microsoft.Azure.NotificationHubs.csproj

```diff
@@ -5,8 +5,9 @@
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <VersionPrefix>4.0.0.0</VersionPrefix>
+    <VersionPrefix>4.1.0.0</VersionPrefix>
     <PackageId>Microsoft.Azure.NotificationHubs</PackageId>
+    <PackageVersion>4.1.0-preview1</PackageVersion>
```

Comment:
Is this intended to be a preview?

Reply:
No, this should be non-preview, will update

Reply:
Fixed